audio-worklet/basic/js/noise-generator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+registerProcessor('noise-generator', class Noise extends AudioWorkletProcessor {
+  static get parameterDescriptors () {
+    return [{
+      name: 'gain',
```

Comment:
Is "gain" the typical name for the amplitude of a noise generator?  That's not my expectation.